src/test/scala/is/hail/expr/ir/IRSuite.scala

```diff
@@ -107,11 +107,13 @@ class IRSuite extends TestNGSuite {
   }
 
   @Test def testArraySort() {
-    assertEvalsTo(ArraySort(NA(TArray(TInt32()))), null)
+    assertEvalsTo(ArraySort(NA(TArray(TInt32())), True()), null)
```

Comment:
This is tested in ArrayFunctions. Might be better to delete it here.

Reply:
There are a few cases like `ArraySort`/`sort`, and `ToDict`/`toDict`, where we have an IR function which is an alias for an IR node. It seems okay to me for those to get tested twice, once in the appropriate functions suite and once in IRSuite. But I'm open to discussion.